File: app/src/main/java/com/aayush/filmipedia/model/GenreResponse.java

package com.aayush.filmipedia.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class GenreResponse {
    @Expose
    private Long page;

    @Expose
    private List<Genre> genres;

    @SerializedName("total_pages")
    private Long totalPages;

    @SerializedName("total_results")
    private Long totalResults;

    public Long getPage() {
        return page;
    }

    public void setPage(Long page) {
        this.page = page;
    }

    public List<Genre> getGenres() {
        return genres;
    }

    public void setGenres(List<Genre> genres) {
        this.genres = genres;
    }

    public Long getTotalPages() {
        return totalPages;
    }

    public void setTotalPages(Long totalPages) {
        this.totalPages = totalPages;
    }

    public Long getTotalResults() {
        return totalResults;
    }

    public void setTotalResults(Long totalResults) {
        this.totalResults = totalResults;
    }

}

File: app/src/main/java/com/aayush/filmipedia/model/GenreResponse.kt
package com.aayush.filmipedia.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class GenreResponse(
    @Expose var page: Long? = null,
    @Expose var genres: List<Genre>? = null,
    @SerializedName("total_pages") var totalPages: Long? = null,
    @SerializedName("total_results") var totalResults: Long? = null
)